central/config/service/service.go

```diff
@@ -118,6 +119,17 @@ func (s *serviceImpl) PutConfig(ctx context.Context, req *v1.PutConfigRequest) (
 	if req.GetConfig() == nil {
 		return nil, errors.Wrap(errox.InvalidArgs, "config must be specified")
 	}
+	if req.GetConfig().GetPrivateConfig() == nil {
```

Comment:
nit:  Should we just move these nil checks into the `validateExceptionConfigReq`?

Reply:
This function receives the full config object which includes retention config, public config, etc. I was surprised to see that there is no validation done within this grpc function. I added a very basic check to ensure that at least private config is not nil to ensure that at least the zero value of the field is received.